### PR TITLE
fix build error

### PR DESCRIPTION
**Pull request by kyrregjerstad**
fixes the Vite build error

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -119,13 +118,13 @@ const routerSwitch = () => {
     // Create offer UI settings
     case 'offerPage':
       document.querySelector('title').innerText = defaultTitle + ` || Create Offer`;
-      setCreateOfferListener();
+      // setCreateOfferListener();
```
Comment (Sovak3441) [inline index.js:121]: Do we need to keep commented out code, since GH already keep track of changes ?

File: index.js [hunk 2]
```diff
@@ -119,13 +118,13 @@ const routerSwitch = () => {
...
       break;
 
     // Apply UI settings
     case 'applyPage':
       document.querySelector('title').innerText = defaultTitle + ` || Apply`;
-      postApplication();
+      // postApplication();
```
Comment (Sovak3441) [inline index.js:127]: Do we need to keep commented out code, since GH already keep track of changes ?

**Review by Sovak3441 — APPROVED** [review]
Left a few inline comments about commented out code

File: index.js (hunk 2 above)
Comment (kyrregjerstad) [reply index.js:127]: Agreed, let's remove them! Taken care of in: https://github.com/NoroffFEU/agency.noroff.dev/pull/754

**Pull request merged**